docker/compose/movement-full-node/docker-compose.telemetry.yml

```diff
@@ -0,0 +1,78 @@
+services:
+  prometheus:
+    image: prom/prometheus:v2.49.1
+    container_name: movement-prometheus
+    command:
+      - "--config.file=/etc/prometheus/prometheus.yml"
+      - "--storage.tsdb.path=/prometheus"
+      - "--web.console.libraries=/usr/share/prometheus/console_libraries"
+      - "--web.console.templates=/usr/share/prometheus/consoles"
+    ports:
+      - "9090:9090"
+    volumes:
+      - ./prometheus.yml:/etc/prometheus/prometheus.yml
+      - prometheus_data:/prometheus
+    healthcheck:
+      test: wget --no-verbose --tries=1 --spider http://localhost:9090/-/healthy || exit 1
+      interval: 5s
+      timeout: 3s
+      retries: 3
+
+  grafana:
+    image: grafana/grafana:10.2.3
+    container_name: movement-grafana
+    ports:
+      - "3000:3000"
+    volumes:
+      - ./grafana/provisioning:/etc/grafana/provisioning
```

Comment:
@nicolaslpf will these values be the same in prod? Or do you imagine we will need some `telelmetry.local.yml` and a `telemetry.production.yml`?